test/application_system_test_case.rb

```diff
@@ -20,49 +21,43 @@
   end
 end
 
-Capybara.register_driver :headless_chrome do |app|
-  Capybara::Selenium::Driver.new(
+Capybara.register_driver :headed_chrome do |app|
+  Capybara::Playwright::Driver.new(
     app,
     browser: :chrome,
-    options: Selenium::WebDriver::Chrome::Options.new.tap do |opts|
-      opts.add_argument "--headless=new"
-      opts.add_argument "--no-sandbox"
-      opts.add_argument "--disable-gpu"
-      opts.add_argument "--disable-dev-shm-usage"
-      opts.add_argument "--window-size=1400x1800"
-
-      # This configurable is used in the Nix development set up, see NIX.md.
-      # It's safe to remove if the Nix pieces are ever ripped out.
-      if ENV.has_key?("SELENIUM_CHROME_BINARY")
-        opts.binary = ENV["SELENIUM_CHROME_BINARY"]
-      end
-    end
+    headless: false
   )
 end
 
-Capybara.register_driver :headless_chrome_in_container do |app|
-  Capybara::Selenium::Driver.new(
+Capybara.register_driver :headless_chrome do |app|
+  Capybara::Playwright::Driver.new(
     app,
-    browser: :remote,
-    url: "http://selenium_chrome:4444/wd/hub",
-    options: Selenium::WebDriver::Chrome::Options.new.tap do |opts|
-      opts.add_argument "--headless=new"
-      opts.add_argument "--disable-gpu"
-      opts.add_argument "--window-size=1400x1800"
-    end
+    browser: :chrome,
+    headless: true,
+    # We only use this env var for the Nix dev environment. For other platforms
+    # we can let playwright use its own downloaded browser.
+    executablePath: ENV.fetch("PLAYWRIGHT_CHROME_BINARY", nil)
   )
 end
 
-Capybara.register_driver :chrome_in_container do |app|
-  Capybara::Selenium::Driver.new(
-    app,
-    browser: :remote,
-    url: "http://selenium_chrome:4444/wd/hub",
-    options: Selenium::WebDriver::Chrome::Options.new.tap do |opts|
-      opts.add_argument "--window-size=1400x1800"
-    end
-  )
-end
+# Capybara.register_driver :headless_chrome_in_container do |app|
+#   Capybara::Playwright::Driver.new(
+#     app,
+#     browser: :remote,
+#     url: "http://selenium_chrome:4444/wd/hub"
+#   )
+# end
+
+# Capybara.register_driver :chrome_in_container do |app|
+#   Capybara::Selenium::Driver.new(
+#     app,
+#     browser: :remote,
+#     url: "http://selenium_chrome:4444/wd/hub",
+#     options: Selenium::WebDriver::Chrome::Options.new.tap do |opts|
+#       opts.add_argument "--window-size=1400x1800"
+#     end
+#   )
+# end
```

Comment:
Do we want to keep this as comments in case there are some bumps down the road with this transition? I lean towards just deleting it and pulling it from git if it becomes important later on.

Reply:
Agreed! This was leftover from Jim's initial experimentation I think. Will clean up.

Reply:
Done in c9b6e3d547aa95e45348251ce1eca594bee7b2f5